src/client/components/Comments/Comment.js

```diff
@@ -235,7 +237,9 @@ class Comment extends React.Component {
             <i className="iconfont icon-offline" />
           )}
         </span>
-        <Avatar username={comment.author} size={comment.depth === 1 ? 40 : 32} />
+        <Link to={`/@${comment.author}`} style={{ height: avatarSize }}>
```

Comment:
I believe there is better way to prevent this link from taking entire height than that.
@jm90m

Reply:
you did the right thing here with setting the height of this, there is no way to disable height with flex box, it will always be 100% of the container, unless you change the flex box direction to column in the parent container but I don't think you would want that in this situation. what you can do is set `max-height` to `avatarSize` as well but its essentially the same thing as setting `height` to `avatarSize` when dealing with flex box.